src/main/java/uk/gov/hmcts/reform/sendletter/controllers/SendLetterController.java

```diff
@@ -56,4 +60,17 @@ public SendLetterController(
 
         return ok().body(messageId);
     }
+
+    @PutMapping(path = "/{id}/sent-to-print-at")
+    @ApiOperation(value = "Update when letter was sent to print")
+    public ResponseEntity<Void> updateSentToPrint(
+        @PathVariable("id") String id,
+        @RequestBody LetterSentToPrintAtPatch patch,
```

Comment:
`SentToPrintAtPatch` would be enough. But (taking into account previous comment) `SentToPrintPatch` looks and sounds better. Personally